NAMESPACE

```diff
@@ -18,6 +16,7 @@ export(set_pins_api_key)
 export(set_sis_db_password)
 export(set_sis_db_username)
 export(uth_clean_student_id)
+export(uth_get_data_from_pins)
```

Comment:
Can we change the name of this function uth_get_data_from_pin? We don't want to switch back and forth between plural and singular.

Reply:
I change the name of the function in issue #3